<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1180</width>
    <height>849</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QTableWidget" name="tableWidget_indices">
   <property name="geometry">
    <rect>
     <x>5</x>
     <y>10</y>
     <width>1171</width>
     <height>421</height>
    </rect>
   </property>
   <column>
    <property name="text">
     <string>Año</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>Mes</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>IRPH Entidades</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>IRPH Bancos</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>IRPH Cajas</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>Euribor</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>Fijo</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>Diferencial</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>Otro</string>
    </property>
   </column>
  </widget>
  <widget class="QTableWidget" name="tableWidget_2">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>440</y>
     <width>1171</width>
     <height>331</height>
    </rect>
   </property>
   <column>
    <property name="text">
     <string>Año</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>Interes Legal</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>Interes demora</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>Fecha inicial</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>Fecha final</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>Observación</string>
    </property>
   </column>
  </widget>
  <widget class="QPushButton" name="pushButton_1">
   <property name="geometry">
    <rect>
     <x>1070</x>
     <y>800</y>
     <width>93</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>SALIR</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_2">
   <property name="geometry">
    <rect>
     <x>770</x>
     <y>800</y>
     <width>251</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>EXPORTAR HISTORICO INTERES LEGAL</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_3">
   <property name="geometry">
    <rect>
     <x>480</x>
     <y>800</y>
     <width>221</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>EXPORTAR HISTORICO INDICES</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
